online_meetings/report.py

```diff
@@ -1,17 +1,19 @@
 import requests
+from requests.auth import AuthBase
```

Comment:
Nitpick: these imports could be tidied up a bit.

Reply:
Okay, doing this

Reply:
We should *probably* use some auto formatter. It doesn't look like Black covers this yet. This is probably tricky because PEP8 recommends you order it by

* Standard Libraries
* 3rd party libraries
* Local Libraries

Looks like isort might take care of this! https://pypi.org/project/isort/ I ran it on all the code in online_meetings.

Reply:
Yeah, I follow that ordering and label as such, though I didn't know PEP8 recommended that, I don't think. Implementing an auto-formatter could be a separate issue.